modules/apps/portal-security/portal-security-permission-impl/src/main/java/com/liferay/portal/security/permission/internal/InlineSQLHelperImpl.java

```diff
@@ -418,10 +418,8 @@ private <T extends Table<T>> Predicate _getPermissionPredicate(
 
 		T table = classPKColumn.getTable();
 
-		Column<T, Long> userIdColumn = table.getColumn("userId", Long.class);
-
 		DSLQuery resourcePermissionDSLQuery = _getResourcePermissionQuery(
-			permissionChecker, modelClassName, userIdColumn, groupIds);
+			permissionChecker, modelClassName, groupIds);
```

Comment:
@evibudai, please place **T table = classPKColumn.getTable();** to https://github.com/liferay-core-infra/liferay-portal/blob/58592900a29f25e693ab25adc86e70e12adabc35/modules/apps/portal-security/portal-security-permission-impl/src/main/java/com/liferay/portal/security/permission/internal/InlineSQLHelperImpl.java#L467.

Reply:
@yuhai it's done